### ISSUE TITLE
A11y_Azure Tools for VS Code [BM]_Default Screen_Screen Reader: VoiceOver is reading as "Skip for No" for "Skip for Now" button

### ISSUE THREAD

**Issue by Ashwinipola26** [microsoft/vscode-azurefunctions#2687]
### Actual Result:
VoiceOver is reading as "Skip for No" for "Skip for Now" button

### Expected Result:
VoiceOver should be reading as "Skip for Now" for "Skip for Now" button

### GitHubTags:
#Azure Tools for VS Code;#A11ySev2; #A11yTCS;#BM-Azure Tools for VS Code-MAC-Feb2021;#DesktopApp;#WCAG2.5.3;#MAC; #ScreenReader;#VoiceOver;#Element:Button;#AINotInScope;#Label in Name

### OS Details:
macOS Catalina Version 10.15.7

### Environment Details:
Application Name: Azure Tools for VS Code
Version: 1.53.2
Commit: 622cb03f7e070a9670c94bae1a45d78d7181fbd4
Date: 2021-02-11T11:45:54.515Z(6 days ago)
Electron: 11.2.1
Chrome: 87.0.4280.141
Node.js: 12.18.3
V8: 8.7.220.31-electron.0
OS: Darwin X64 19.6.0
Azure Function Version: 1.2.1

### Repro Steps:

1. Start VoiceOver
2. Open Visual studio code
3. Tab till extension button and press enter on it.
4. Search for "Azure Functions" and select it.
5. Tab till "Install" button and press enter on it.
6. Tab till "Azure" button from left side pane and press enter on it.
7. Default screen will get open.
8. Tab till Create a Project button and press enter
9. Tab till Select the folder that will contain your function project and select any recently used
10. Tab till Select a Language and select any one language
11. Tab till Skip for now list item under "Select a template for your Project's first function and check whether VoiceOver is reading Skip for no or Skip for Now

### User Impact
Screen Reader users will get confuse if VoiceOver is reading "Skip for No" for "Skip for Now" button name

### Attachment for Reference:
[Bug_2687_VoiceOver is reading Skip for Now as "Skip for No".zip](https://github.com/microsoft/vscode-azurefunctions/files/6001260/Bug_2687_VoiceOver.is.reading.Skip.for.Now.as.Skip.for.No.zip)

**Comment by ejizba** [comment]
This is not unique to our extension. Closing in favor of upstream VS Code issue: https://github.com/microsoft/vscode/issues/120992
